lib/ansible/modules/cloud/scaleway/scaleway_security_group_rule.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+#
+# Scaleway Security Group Rule management module
+#
+# Copyright (C) 2018 Antoine Barbare (antoinebarbare@gmail.com).
+#
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: scaleway_security_group_rule
+short_description: Scaleway Security Group Rule management module
+version_added: "2.8"
+author: Antoine Barbare (@abarbare)
+description:
+    - This module manages Security Group Rule on Scaleway account
+      U(https://developer.scaleway.com)
+extends_documentation_fragment: scaleway
+
+options:
+  state:
+    description:
+      - Indicate desired state of the Security Group Rule.
+    default: present
+    choices:
+      - present
+      - absent
+
+  region:
```

Comment:
Since a rule belongs to a security group and that security group belongs to a region, could not region from security group reused and this parameter be deleted ? Or should the region parameter be removed from the security group module (because security_group_facts module doesn't display any region) ?

Reply:
Actually, we need to get this information from the user to build the api object which is related to a region. For now, we have only two regions, it's not really a problem and we could try each one but with more regions, this behaviour is not really optimal.